admin/services/data-access/sql.service.js

```diff
@@ -288,6 +318,44 @@ sqlService.query = async (sql, params = [], redisKey) => {
   return retry(query, retryConfig, dbLimitReached)
 }
 
+/**
+ * Query data from SQL Server over a readonly connection
+ * @param {string} sql - The SELECT statement to execute
+ * @param {array} params - Array of parameters for SQL statement
+ * @param {string} redisKey - Redis key to cache resultset against
+ * @returns {Promise<*>}
+ */
+sqlService.readonlyQuery = async (sql, params = [], redisKey) => {
+  // logger.debug(`sql.service.readonlyQuery(): ${sql}`)
```

Comment:
I guess these debug logs need to be removed

Reply:
Yeah, they were preserved in case we wanted them at some point,  but a sql specific category would be cleaner